src/main/scala/nl.knaw.dans.easy.solr4files/CommandLineOptions.scala

```diff
@@ -59,8 +59,8 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
     footer(SUBCOMMAND_SEPARATOR)
   }
   val delete = new Subcommand("delete") {
```

Comment:
@PaulBoon @janvanmansum @lindareijnhoudt would it make sense to have an _'Are you sure?'_ message for this `delete` command?

Reply:
A rest request (though _not yet_ implemented as such) asking for confirmation?

Reply:
For a command line interface this makes sense, but then I would also like to have a '--force' option to bypass it, which is useful when scripting it. 

But more importantly I don't think it has a high priority. Would users do a delete by accident, it's not a button you accidentally click. And even if you do regret your delete, you can reindex it.

Reply:
agreed! The command line would have an 'are you sure'. I also like the `--force` for bypassing this! Of course the REST interface will not ask for confirmation. This is a **machine-2-machine** interface, so it is less likely to go wrong. The 'are you sure' and `--force` are there to prevent **human** error!